apps/explorer/src/components/Layout/index.tsx

```diff
@@ -7,99 +7,104 @@ import {
 } from '@growthbook/growthbook-react';
 import { RpcClientContext } from '@mysten/core';
 import { WalletKitProvider } from '@mysten/wallet-kit';
+import * as Sentry from '@sentry/react';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
-import { Fragment, useMemo } from 'react';
+import { useEffect, useMemo } from 'react';
 import { Toaster } from 'react-hot-toast';
 import { Outlet, ScrollRestoration } from 'react-router-dom';
 
 import { usePageView } from '../../hooks/usePageView';
 import Footer from '../footer/Footer';
 import Header from '../header/Header';
 
-import { NetworkContext, useNetwork } from '~/context';
+import { useNetwork } from '~/hooks/useNetwork';
 import { Banner } from '~/ui/Banner';
 import { DefaultRpcClient } from '~/utils/api/DefaultRpcClient';
 import { growthbook } from '~/utils/growthbook';
 import { queryClient } from '~/utils/queryClient';
 
 export function Layout() {
-    const [network, setNetwork] = useNetwork();
+    const [network] = useNetwork();
     const jsonRpcProvider = useMemo(() => DefaultRpcClient(network), [network]);
     const networkOutage = useFeatureIsOn('explorer-network-outage');
 
+    useEffect(() => {
```

Comment:
I think I need to test this a little more... this was previously in a `useLayoutEffect`, so I want to make sure we aren't running into a issue with Growthbook where feature targeting doesn't work correctly